includes/tracks/events/class-wc-products-tracking.php

```diff
@@ -38,6 +39,18 @@ public function track_product_updated( $product_id, $post ) {
 		);
 
 		WC_Tracks::record_event( 'product_edit', $properties );
+
+		$product = new WC_Product( $product_id );
+		$update_properties = array(
+			'product_id'            => $product_id,
+			// 'product_type'          => TODO where does this come from?? there is no product type, only category ids.
```

Comment:
@pmcpinto There is no product type on a product, only the category IDs used for the product taxonomy. What should be used here?

Reply:
Hey @becdetat, my goal was to understand what type of product they're adding (simple product, grouped product, external/affiliate product, variable product) and if they're tagged as virtual and/or downloadable. Do we have a way to track this as an event prop?
I'm not sure if we can get this through category ID.

Reply:
Ok, I see. I think most of those are not mutually exclusive so it would still have to be an array, but it should be possible to do this.

Reply:
Actually on second thought an event property per dimension (eg. `simple_or_grouped == 'simple'`, `affiliate == 'yes'`) would probably be easier to both build and query.